app/models/validation_request.rb

```diff
@@ -9,8 +9,17 @@ module ValidationRequest
     aasm.attribute_name :state
 
     aasm do
-      state :open, initial: true
+      state :pending, initial: true
+      state :open
       state :closed
+
+      event :mark_as_sent! do
+        transitions from: :pending, to: :open
```

Comment:
so each validation request goes from a default event of pending to open... but could you help me understand where we are updating the requests to closed using the state machine? is it the helpers?

Reply:
so yeah I guess I forgot this bit... I wasn't sure where we do close them. The only trace I can find is:

```

Reply:
yes indeed - validation_requests already had a state of "closed" that gets set whenever a reply from the customer is hit through the api - so we could tie the state machine transition to that I suppose